lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+  attr_accessor :menu_dishes
+  def initialize
+    @menu_dishes = { :pizza => 4.00, :sausage => 2.00, :curry => 6.50,
+      :rice => 2.00, :chilli => 6.00, :salad => 3.50 }
+  end
+
+  def list_menu
+    @menu_dishes
```

Comment:
A very small thing, but the fact that you have an accessor for your @menu_dishes instance variable means that you don't need the `@` here